fairscale/optim/oss.py

```diff
@@ -91,6 +104,98 @@ def state_dict(self) -> dict:
         """ Gets this rank's state_dict. """
         return self.optim.state_dict()
 
+    def _collect_state_dict(self) -> List[dict]:
+        """
+        Collect all the state shards
+        """
+        empty_buffer = torch.empty([1], dtype=torch.uint8)
+        global_optim_state = []
+        local_state = self.state_dict()
+
+        if len(local_state["state"]) == 0:
+            return []
+
+        for rank in range(dist.get_world_size(group=self.group)):
+            if rank == self.rank:
+                logging.info("Saving self state")
```

Comment:
Should the logs be debug level? I don't know what the convention is but worry about oss being too chatty.